uiowa/uiowa
# I2457 - student profiles

## Testing Instructions
1. `blt ds --site=admissions.uiowa.edu` & `blt frontend`
2. Add a 'Student Profile' -> need to create at least two different profiles w/ diff hometown info
2a. One student profile with hometown, state and United states as country -> make sure displays only hometown and abbreviated state value (not country if country = US)
2b. One student profile with hometown and country (other than US) -> make sure displays hometown and full name of country
2c. When remove hometown value, state, and country -> font awesome icon should not display
3. Once added, make sure the full node view contains a 'Student profiles' view block on right column with the profile images of existing/published student profiles 
3a. Make sure grid display of 'Student profiles' renders correctly.. grid = 3 x 3 -> add at least four profiles to make sure fourth wraps around
3a.  Make featured image diff from profile image (profile image will display on student profile grid displayed in full node)
3b. Each student profile content will add an image to the student profiles grid
4. Go to http://admissions.local.drupal.uiowa.edu/student-profiles and make sure teasers render correctly
5. If area of study isn't populated, the book font awesome icon should not be displayed; if area of study is populated, book icon should display before visually hidden label. 

Closes #2457, closes https://github.com/uiowa/uiowa/issues/2458, and closes https://github.com/uiowa/uiowa/issues/2409.
